newt/cli/target_cmds.go

```diff
@@ -684,7 +704,7 @@ func AddTargetCommands(cmd *cobra.Command) {
 
 	setCmd := &cobra.Command{
 		Use: "set <target-name> <var-name>=<value>" +
-			"[:<var-name>=<value>...]",
+			"[<var-name>=<value>...]",
```

Comment:
Good catch.  However, there is still a minor issue here.  To be pedantically correct, there should be a space after the first '>' (i.e., before '[').